### DISCUSSION TITLE
Is there a Dark Mode?

### DISCUSSION THREAD

**Discussion by d7eeem** [rogerfar/rdt-client#601] [Q&A] [answered]
I use Dark reader ext for browser but recently it stopped working and i didn't notice that RDT doesn't have a dark mode any help would be appreciated

**Comment by d-allin** [comment, marked as answer]
Dark reader works fine.

If you're accessing things locally rather than via domain and want it on for one service but off for others, use the site list. For example:

Invert Listed Only:
XX.XX.XX.XXX:6500
XX.XX.XX.XXX:8020

Not Invert Listed:
XX.XX.XX.XXX

**Reply by d7eeem** [reply]
Firefox is finicky dark reader works for RDTC on brave and Chromium based browsers.

**Reply by dragged9698** [reply]
> Firefox is finicky dark reader works for RDTC on brave and Chromium based browsers.

dark reader works great for me on firefox, not sure what issue you're facing
